content/en/docs/hardware/v00.03/build-instructions/index.de.md

```diff
@@ -330,15 +329,15 @@ Das fertig bestückte PCB eines OBS v00.03.
 * Gewinkelte Stiftleiste mit 4 Pins vorbereiten
 * Stiftleiste in OBS-PCB verlöten (Unterseite, langes Ende der Pins seitlich herausschauend)
 * Eng anliegend verlöten
-* GPS-Modul aufstecken 
+* GPS-Modul aufstecken
 * Verlöten und Stifte kürzen
 
 * **Achtung!** Achte darauf, dass das kurze Ende der Stiftleiste in das Board und das lange Ende in das GPS-Modul eingelötet wird. Ansonsten ragt das GPS-Modul aus dem Gehäuse heraus.
 
 {{< slider >}}
   {{< slider-image
     src="PCB00.03.10_13_GPS-Module_01_Modul.jpg"
-    alt="Das GPS-Modul mit Antenne (große Variante)" >}}
+    alt="Das GPS-Modul mit Antenne (große Variante). Überstehenden Teil der Platine abknicken und entfernen." >}}
```

Comment:
Hast du probiert ob der Text noch bequem in den Caption-Bereich des Sliders passt, evtl auch auf mobil? Könnte eng werden, evtl sollte das einfach ein einfacher Stichpunkt oben sein.

Reply:
Ja, habe ich, funktioniert bei mir. Allerdings wird der alt-Text generell immer zweimal angezeigt, was ich sehr sonderbar finde. Da das jedoch vorher auch so war, habe ich nichts unternommen es zu ändern. Da ich hier neu bin, wollte ich euch nicht gleich mit Issues fluten.

Reply:
Habe das noch kurz angesehen: Das Verhalten kommt von 

> captionContent += '<div class="iis-caption-title">' + slide.getAttribute('title') + '</div>'; 

aus iis-captions.js  von assets/js/slider.

Die Datei wurde seit 2015 nicht angepasst: https://github.com/Codeinwp/Ideal-Image-Slider-JS/blob/master/extensions/captions/iis-captions.js